docs/source/json-api/index.rst

```diff
@@ -803,12 +803,19 @@ Nonempty HTTP Response with Unknown Template IDs Warning
         "status": 200
     }
 
-Fetch All Known Parties
-***********************
+Fetch Parties by Identifiers
+****************************
 
 - URL: ``/v1/parties``
-- Method: ``GET``
-- Content: <EMPTY>
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    ["Alice", "Bob", "Dave"]
+
+If empty JSON array is passed: ``[]``, this endpoint will return all known parties.
```

Comment:
I'm not convinced that is a good idea. I prefer staying consistent and returning an empty array in that situation. If we want to be able to get all parties on the POST endpoint, we should signal this by sending no content at all. But do we actually want this?

Reply:
all right, we will return only parties specified in the input. Empty input array would mean empty array in the output.

Reply:
we actually expect a NonEmptyList of party ids now, if empty array passed, the current implementation returns a documented error:
```
{
  "status": 400,
  "errors": [
    "JsonReaderError. Cannot read JSON: <[]>. Cause: spray.json.DeserializationException: must be a list with at least 1 element"
  ]
}
```
I think error makes more sense, why would you ask for an empty array.

Reply:
Erroring out on the empty list is fine too.